.github/workflows/stale_issue_pr.yaml

```diff
@@ -0,0 +1,35 @@
+name: 'Stale issue & PR handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@main
+        id: stale
+        with:
+          ascending: true
+          close-issue-message: 'Issue closed due to inactivity.'
+          close-pr-message: 'Pull request closed due to inactivity.'
+          days-before-close: 10
+          days-before-stale: 30
+          stale-issue-label: stale
+          stale-pr-label: stale
+          # Not stale if have this labels 
+          exempt-issue-labels: 'bug'
+          exempt-pr-labels: 'bug'
+          operations-per-run: 100
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it has been open 30 days
+            with no activity. Remove stale label or comment or this issue will be closed in 10 days
+          stale-pr-message: |
+            This PR has been automatically marked as stale because it has been open 30 days
+            with no activity. Remove stale label or comment or this PR will be closed in 10 days
+      - name: Print outputs
```

Comment:
What's this step for?

Reply:
removed, was looking at vpc cni outputs but was looking at the wrong stage, it's printing empty arrays. removed!